src/core/components/SiteNotices/index.js

```diff
@@ -24,6 +24,15 @@ type InternalProps = {|
   i18n: I18nType,
 |};
 
+// This is needed because of https://github.com/mozilla/addons-frontend/issues/8616
+//
+// We cannot use `sanitizeUserHTML()` on a `<span />`, which is required to
+// avoid the UI glitch so we configure our own sanitize function to make sure
+// it is safe to use `<span />`.
+const sanitizeNoticeHTML = (text: string): string => {
+  return sanitizeHTML(nl2br(text), ['a', 'b', 'br', 'em', 'i', 'strong']);
```

Comment:
I wonder if we really have a use-case for presentational markup such as `i` and `b` in place of `em` and `strong`, buy that's something that can be discussed independently of this patch.

Reply:
I think it is convenient for admins, especially since there is no WYSIWYG.